pkgs/fc/agent/fc/maintenance/tests/test_maintenance.py

```diff
@@ -203,10 +210,15 @@ def test_request_update_unchanged_system_and_no_other_requests_skips_request(
         "fc.maintenance.maintenance.UpdateActivity.from_enc", from_enc_mock
     )
 
+    monkeypatch.setattr(
+        "fc.util.nixos.get_free_store_disk_space", lambda *a: 10 * 1024**3
```

Comment:
This monkeypatching *could* be put into a separate fixture that is just pulled in by all other tests and fixtures that run in an environment with enough free space. That reduces unnecessary code verbosity.

Reply:
I thought about this but decided to keep it. Tests can be a more explicit and copying around two one-liners is ok, I think.